services/app-web/src/pages/App/AppRoutes.tsx

```diff
@@ -216,7 +232,13 @@ const CMSUserRoutes = ({
                     )}
                     <Route
                         path={RoutesRecord.SUBMISSIONS_SUMMARY}
-                        element={<SubmissionSummary />}
+                        element={
```

Comment:
@haworku @macrael should the CMS version of the submissionSummary use the V2 page? They have the 'unlock contract' button that isn't currently covered

Reply:
Ugh! Scoop creep. Make it a new ticket. Yes it should because thats the only way CMS can see linked rates stuff. 

Maybe we could get away fetching with new API and doing the unlock with the old HPP API and the old unlock on this v2 page for this one case on the V2 page. Need to look

Reply:
I created the ticket [here](https://jiraent.cms.gov/browse/MCR-4052)